conf/switches.conf.example

```diff
@@ -21,7 +22,7 @@ isolationRole = isolation
 macDetectionRole = macDetection
 voiceRole = voice
 inlineRole = inline
-VoIPEnabled = no
+VoIPEnabled = N
```

Comment:
This is what I mean :

In switches.conf :
```
[default]
param1=value1
```

In switches.conf.defaults :
```
[default]
param1=value69
param2=value2
```

We need to have the following result : 
```
[default]
param1=value1
param2=value2
```

I am asking you because the ConfigStore and pf::config::cached are doing a lot of magic above the Config::IniFiles library and you are the only one that knows how it works. So I would expect you to shed some light on this